README.md

```diff
@@ -5,8 +5,16 @@ There is some environment configuration and addtional files needed for this app
 
```

Comment:
I feel instead of having the `.env` the configuration in README create a file`sample.env` default to Local setting

Reply:
Great, I created .env.sample instead of this

Reply:
+1!